README.rst

```diff
@@ -94,6 +94,19 @@ Volto Blocks Support
 plone.volto enables the new Volto Blocks editor on ``Document``, ``Language Root Folder`` and ``Site Root``.
 
 
+Block Types Index
+-----------------
+
+plone.volto adds a ``block_types`` index to the Plone catalog.
+It can be used to query for items that use a particular type of block::
```

Comment:
I prefer explicit lexers to implicit, which this and the next suggestion do.

```suggestion
It can be used to query for items that use a particular type of block.
```